### ISSUE TITLE
Problem with terminal_services collector

### ISSUE THREAD

**Issue by S3maks** [prometheus-community/windows_exporter#1086]
Good afternoon. There is a problem with the collector.
The collector is needed to monitor RDP sessions on Windows.
Under load, when users start their working day, the windows exporter loses contact with prometheus, although in the services it says that the exporter is running.
End point returns "server returned HTTP status 500 internal server error".
Exporter gives 15 errors before shutting down (see screenshot)
![forgit](https://user-images.githubusercontent.com/112616692/199005191-912d1382-a2bf-408e-bf2c-9cecff46ba9e.jpeg)

This problem is observed with the 19th and 20th version exporter on version Win 2012r2
The failure occurs when the load increases, when approximately 80-90 users are connected via RDP, once a day

Without terminal_services, the collector exporter works stably.
What could be the problem  ?

**Comment by Outek** [comment]
We have the same issue, it begins around 60-70 concurrent sessions.
Windows Server 2019

**Comment by anthony2611** [comment]
We also have the same issue, but we have only 20 users 
Windows Server 2022

**Comment by breed808** [comment]
Assuming the issue is still occurring, you can identify if Perflib is returning duplicate metrics by querying the `Terminal Services Session` CounterSet:

```powershell
# List Counter Sets (confirm if "Terminal Services Session" CounterSet is present)
Get-Counter -ListSet * | Sort-Object -Property CounterSetName | Select CounterSetName
# List counters for set
Get-Counter -ListSet 'Terminal Services Session' 
# Get a counter from the set
Get-Counter -Counter '\Terminal Services Session(*)\Handle Count'
```

The output from the last command should identify if duplicate metrics are being presented to windows_exporter.
Alternatively you can run the [perflib_exporter](https://github.com/leoluk/perflib_exporter) directly to gain additional information on the metrics being presenting to windows_exporter.

**Comment by Outek** [comment]
fixed with #1184 ?

**Comment by S3maks** [comment]
> fixed with #1184 ?

We will check, thanks

**Comment by S3maks** [comment]
Problem solved
thanks
